test/Libraries/Microsoft.Extensions.Diagnostics.ResourceMonitoring.Tests/CalculatorTests.cs

```diff
@@ -192,4 +193,29 @@ public void OverUtilized()
         Assert.Equal(1500UL, record.MemoryUsedInBytes);
         Assert.Equal(1000UL, record.SystemResources.GuaranteedMemoryInBytes);
     }
+
+    /// <summary>
+    /// Ensure that CPU stats work appropriately.
+    /// </summary>
+    [Fact]
+    public void DoubleCpuUnits_WithFullyUtilizaed()
+    {
+        var limitedResources = new SystemResources(2, 2, TotalMemoryInBytes, TotalMemoryInBytes);
+
+        var secondSnapshotTimeSpan = _firstSnapshot.TotalTimeSinceStart.Add(TimeSpan.FromSeconds(5));
```

Comment:
Ditto, we use `var` only if the type is obvious.

Reply:
All style issue in this file fixed